### PR TITLE
refactor: fix typos

### PR DESCRIPTION
**Pull request by wizard-28**
This aims to fix majority of the typos throughout the code base. Approximately ~~300~~ 200 files will be edited when this PR is finalized.

### CODE REVIEW

File: kernel-open/common/inc/nv_uvm_interface.h [hunk 1]
```diff
@@ -121,7 +121,7 @@ NV_STATUS nvUvmInterfaceSessionDestroy(uvmGpuSessionHandle session);
     nvUvmInterfaceDeviceCreate
 
     Creates a device object under the given session for the GPU with the given
-    UUID. Also creates a partition object for the device iff bCreateSmcPartition
```
Comment (Newbytee) [inline kernel-open/common/inc/nv_uvm_interface.h:124]: iff is probably not a typo: https://www.lexico.com/definition/iff

Comment (wizard-28) [reply kernel-open/common/inc/nv_uvm_interface.h:124]: Okay, fixing it

Comment (wizard-28) [reply kernel-open/common/inc/nv_uvm_interface.h]: Reverted it everywhere.

**Comment by Newbytee** [comment]
FYI you don't have to push every single commit individually. You can make all commits locally and then push them when you're done for the day. Creates less spam for people subscribed to this thread.

File: kernel-open/nvidia/nv-acpi.c [hunk 2]
```diff
@@ -424,7 +424,7 @@ static void nv_acpi_event(acpi_handle handle, u32 event_type, void *data)
 
     if (event_type == NVIF_DISPLAY_DEVICE_CHANGE_EVENT)
     {
-        /* We are getting NVIF events on this machine. We arent putting a very
+        /* We are getting NVIF events on this machine. We aren't putting a very
```
Comment (YusufKhan-gamedev) [inline kernel-open/nvidia/nv-acpi.c:427]: :(

Comment (wizard-28) [reply kernel-open/nvidia/nv-acpi.c:427]: Could you elaborate on the your issue with this change?

**Comment by mtijanic** [comment]
Thank you for the PR, but marking as `cosmetic-postponed` for now. Please see https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/190 for details.

**Comment by mtijanic** [comment]
Thank you for the PR, but marking as `cosmetic-postponed` for now. Please see https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/190 for details.

**Comment by niv** [comment]
Thank you for your interest in contributing to open-gpu-kernel-modules.

As recently explained in [CONTRIBUTING.md](https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CONTRIBUTING.md), we have decided to not accept cosmetic changes for now.

All changes going into this repository incur significant overhead on our end, and we'd like to focus our resources on executable code changes; so we have decided to close all PRs that make no code changes.

If and when this position changes, your PR will still be in the archives and revisited (as long as it's still tagged cosmetic-postponed).

Please do not feel discouraged to submit other changes! In addition, we will be happy to keep the conversation going on the Discussion boards here on GitHub, and elsewhere.

**Pull request closed without merging**